### ISSUE TITLE
DOC: Juypter notebook with examples

### ISSUE THREAD

**Issue by karllark** [karllark/dust_extinction#54]
Need more examples of how to use and such examples in juypter notebooks would be ideal.

**Comment by krislars** [comment]
Here's a place to start:
[https://gist.github.com/krislars/7f9ea50109a93d02bde712859fdbd995](https://gist.github.com/krislars/7f9ea50109a93d02bde712859fdbd995)
Suggestions welcome!

**Comment by karllark** [comment]
This is great.  Should we put these somewhere in the dust_extinction package?  Anyone know what the standard practice is?  I can check later this month as I'll be attending the Python in Astronomy meeting in NY.  But if someone else already knows...

**Comment by emirkmo** [comment]
I think the place to have example notebooks would be in the docs somewhere, not in package itself.

**Comment by krislars** [comment]
@karllark, I'm glad that you'll be attending the Python in Astronomy meeting!  

I'm starting to wonder if example notebooks might end up repeating what is already in the docs.  A different approach would be to submit an [Astropy Tutorial](https://github.com/astropy/astropy-tutorials).  There is more narrative encouraged with the tutorials, and could include explanation (as you did in #57) why getting color from spectral extinction is tricky, but give an example of doing it with synthetic photometry from another package.

This is essentially what @eteq said in the discussion at #47, and I think his idea is a good one.

**Comment by emirkmo** [comment]
Very cool! Wish I could have attended.. I don’t mind taking a look at the contributing tutorial and drafting something, or contributing to a draft if  plans. I’ll have time to work on it on May 12 and 13. I’ll check the status of it at that time and see what is needed.

**Comment by karllark** [comment]
Sounds great.  The workbook you shared earlier is a good start.  I did learn they wanted more than just dust_extinction in the tutorial, hence I included steps explicitly using astropy material.  Hopefully this makes sense.  Happy to expand/contribute as appropriate.
One note: I think we should use a spectrum that includes the UV as then we have a feature (2175 A) that will show if the unextinguishing worked or not.  Hard to diagnose this visually without some feature.
